website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
I guess I wanted to make sure that date_modified was definitely set, but I guess the if save: block is saving the new log list on the node only then should date_modified be updated. 

Removed .save() that was outside this block.